src/frameworks/CoreServices/src/LaunchServices/LSInfo.m

```diff
@@ -102,6 +102,15 @@
 	return retval;
 }
 
+// Expected in: /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
+// WARN: Deprecated.
+CFArrayRef LSCopyAllHandlersForURLScheme(CFStringRef inURLScheme)
+{
+	puts("LSCopyAllHandlersForURLScheme STUB");
+	// NULL is a safe assumption to fall back on in the stub, since it means none were found.
+	return NULL;
+}
```

Comment:
Is this how you'd prefer I implement this stub?

Reply:
My concern is that some stubs are checking for an env var before they print.

Reply:
It probably would be better if we did check the environment variable before we print the stub, but I'm not super strict about it. I'm okay with how you handled it.